plugins/veeValidate.ts

```diff
@@ -9,6 +9,15 @@ extend('mustBeUrl', (value: string): boolean => {
   return /^https?:\/\/.+/.test(value);
 });
 
+extend('bothPasswordsSame', {
+  params: ['target'],
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  validate(value, { target }) {
+    return value === target;
```

Comment:
This ts-comment is not ideal, but this is what the docs show to do to ensure that passwords are equal

https://logaretm.github.io/vee-validate/advanced/cross-field-validation.html#targeting-other-fields

Reply:
We should have added a comment there with that before merging